conda/dev-environment-unix.yml

```diff
@@ -26,13 +26,11 @@ dependencies:
  - mdformat>=0.7.17,<0.8
  - ninja
  - numpy
- - pillow
```

Comment:
pillow is used in showgraph, psutil is used a bunch across tests and profiling

Reply:
both of them were duplicates

Reply:
Ah yeah we took a pretty liberal interpretation of alphabetical order I see 😂